api/provider/aws/task_list.go

```diff
@@ -0,0 +1,75 @@
+package aws
+
+import (
+	"github.com/aws/aws-sdk-go/service/ecs"
+	"github.com/quintilesims/layer0/common/models"
+)
+
+func (t *TaskProvider) List() ([]*models.TaskSummary, error) {
+	clusterNames, err := listClusterNames(t.AWS.ECS, t.Config.Instance())
+
+	if err != nil {
+		return nil, err
+	}
+
+	summaries := []*models.TaskSummary{}
+	for _, clusterName := range clusterNames {
+		runningTaskARNs, err := t.listTasksByStatus(clusterName, "RUNNING")
+		stoppedTaskARNs, err := t.listTasksByStatus(clusterName, "STOPPED")
+
+		if err != nil {
+			return nil, err
+		}
+
+		environmentID := delLayer0Prefix(t.Config.Instance(), clusterName)
+
+		for _, taskARN := range append(runningTaskARNs, stoppedTaskARNs...) {
+			summary := models.TaskSummary{
+				TaskID:        *taskARN,
```

Comment:
`taskARN` != `taskID`. Also, use safe deferences from the aws package - I believe for strings it is `aws.StringValue(*string)`